WebCLI/tests/testNewMolecule.py

```diff
@@ -39,12 +39,11 @@ def test_add_molecule_with_incorrect_structure(self):
         result = len(Molecule.objects.filter(name='Lithium hydride2'))
         self.assertEqual(result, 0)
 
-    def test_add_molecule_with_incorrect_orbitals(self):
+    def test_add_molecule_with_no_orbitals(self):
```

Comment:
The old test should not be removed. Incorrect orbitals should still be validated.